package.json

```diff
@@ -39,6 +39,9 @@
     "./src/sizes": "./src/props.sizes.js",
     "./src/svg": "./src/props.svg.js",
     "./src/zindex": "./src/props.zindex.js",
+    "./src/media": "./src/props.media.js",
```

Comment:
this was already exposed as `"./esm/media": "./src/props.media.js",` 👍🏻

Reply:
ooh, i missed that. undone! 👍

if you would like everything to be under `esm/`, i can update the other two as well.

the documentation mentions `src/` fwiw
![](https://user-images.githubusercontent.com/9084735/208190162-97128c9c-900c-42a8-b393-22f2362b5529.png)

Reply:
i'll fix in #327, thanks!

Reply:
> i'll fix in #327, thanks!

that is technically a breaking change, jsyk (after you mentioned `esm/media`, i added it in my script. now if i upgrade to 1.5.2, it will break my script)

Reply:
yeah, it is technically one! but i also silently released it, never documented it and have discord if anyone else finds the feature and change.